ApexOSA/polarimeter.py

```
@@ -0,0 +1,226 @@
+        """
+
+        Values = []
+        if not self.__Simulation:
+            Command = "POLRAWPOWER?\n"
+            Send(self.__Connexion, Command)
+            Str = Receive(self.__Connexion, 64)[:-1]
+
+            for v in Str.split(" "):
+                try:
+                    Values.append(int(v))
+                except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)